integrasjonspunkt/src/main/java/no/difi/meldingsutveksling/nextmove/ServiceBusRestClient.java

```diff
@@ -101,36 +94,44 @@ public void sendMessage(byte[] message, String queuePath) {
 
         URI uri = convertToUri(resourceUri);
         log.debug("Calling {}", resourceUri);
-        ResponseEntity<String> response = restTemplate.exchange(uri, HttpMethod.POST, httpEntity, String.class);
-        if (!asList(HttpStatus.OK, HttpStatus.CREATED).contains(response.getStatusCode())) {
-            log.debug("{} got response {}, returning empty", resourceUri, response.getStatusCode().toString());
+        try {
+            ResponseEntity<String> response = restTemplate.exchange(uri, HttpMethod.POST, httpEntity, String.class);
+            if (!asList(HttpStatus.OK, HttpStatus.CREATED).contains(response.getStatusCode())) {
+                log.debug("{} got response {}, returning empty", resourceUri, response.getStatusCode().toString());
+                return Optional.empty();
+            }
+
+            ServiceBusMessage.ServiceBusMessageBuilder sbmBuilder = ServiceBusMessage.builder();
+            BrokerProperties brokerProperties = getBrokerProperties(response);
+
+            String messageId = brokerProperties.getMessageId();
+            sbmBuilder.lockToken(brokerProperties.getLockToken())
+                    .messageId(messageId)
+                    .sequenceNumber(brokerProperties.getSequenceNumber());
+
+            try {
+                ServiceBusPayload payload = payloadConverter.convert(response.getBody(), messageId);
+                sbmBuilder.payload(payload);
+                log.debug(format("Received message on queue=%s with messageId=%s", localQueuePath, payload.getSbd().getDocumentId()));
+                return Optional.of(sbmBuilder.build());
+            } catch (JAXBException e) {
+                log.error(String.format("Error creating old format from message id=%s", messageId), e);
+            }
+        } catch (ResourceAccessException | IOException e) {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Either log or rethrow this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1166)